website/static/js/fangorn.js

```diff
@@ -2197,7 +2200,8 @@ Fangorn.Utils = {
     openParentFolders : _openParentFolders,
     dismissToolbar : _dismissToolbar,
     uploadRowTemplate : uploadRowTemplate,
-    resolveIconView: resolveIconView
+    resolveIconView: resolveIconView,
+    orderFolder: _fangornOrderFolder
```

Comment:
`_fangornOrderFolder` begins with an underscore, presumably because it wasn't originally intended to be exported outside the scope of `fangorn.js`. Now that it is being exported (and as `orderFolder`), is there any reason not to rename it `orderFolder`?

My rationale for this is that it would make it more clear that those functions defined with an underscore elsewhere are private members of the module, and make it so a text search for the string `"orderFolder"` would return the definition of the function, instead of just its export.

I don't feel that this is a *necessary* change, but if you disagree I'd like to hear your perspective. Steve, if you read this, don't hold up merging this PR on account of this :)

Also, if you do rename it, consider renaming `_openParentFolders` and `_dismissToolbar` as well, for the same reasons. It's not code that's part of this PR, but it would be easy enough to do them all three at once, and I believe would help make this module a bit more consistent.